pkg/apis/deployment/v1/server_group_spec.go

```diff
@@ -70,6 +70,8 @@ type ServerGroupSpec struct {
 	VolumeClaimTemplate *v1.PersistentVolumeClaim `json:"volumeClaimTemplate,omitempty"`
 	// VolumeResizeMode specified resize mode for pvc
 	VolumeResizeMode *PVCResizeMode `json:"pvcResizeMode,omitempty"`
+	// AntiAffinity specified additional antiAffinity settings in ArangoDB Pod definitions
```

Comment:
Should there also be Affinity?

Reply:
I can add also Affinity and Node Affinity  - so it will cover whole functionality

Reply:
Done

Reply:
Done